dmscripts/models/queries.py

```diff
@@ -64,3 +64,10 @@ def add_counts(join, group_by, model, data, directory):
         )['_count'].fillna(0)
 
     return data
+
+
+def expand_json_subfields(field, subfields, data):
```

Comment:
But we don't refer to data as 'parent' anywhere else in either file so then the logical thing to do is to call it `copy_json_subfields_to_parent` but we don't _refer_ to data in any of the other transformation method names.
`copy_json_subfields_to_parent` Sounds overly verbose compared with everything else in that file.